portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,8 +62,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String username = request.getParameter("username");
     String message = request.getParameter("comment-or-question");
-    messages.put(username, message);
-    // messages.add(username);
+    String maxNumCommentsString = request.getParameter("max-num-comments");
+    try {
+      maxNumComments = Integer.parseInt(maxNumCommentsString);
+    } catch (NumberFormatException e) {
+      System.err.println("Could not convert to int: " + maxNumCommentsString);
+      System.err.println("Using default value of 7.");
+      maxNumComments = 7;
+    }
+
+    Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("username", username);
```

Comment:
Constants like "Comment", "username", "message" and "timestamp" are used multiple times. Consider defining a constant for those to make it less error prone.

Reply:
That's a good point, I'll consider it